infrastructure/building-xahau/mac-os-13.5.2.md

```diff
@@ -1,14 +1,22 @@
 # Mac OS - 15.0.1
 
-| Dependency  | Working Version |
-| ----------- | --------------- |
-| Apple Clang | 14.3.1          |
-| LLVM        | 14              |
-| LLD         | 14              |
-| Boost       | 1.86.0          |
-| CMake       | 3.23.1          |
-| Protobuf    | 3.20.0          |
-| WasmEdge    | 0.11.2          |
+| Dependency  | Working Versions |
+|-------------|------------------|
+| Apple Clang | 14.3.1, 16.0.0   |
+| LLVM        | 14               |
+| LLD         | 14               |
+| Boost       | 1.86.0           |
+| CMake       | 3.23.1           |
+| Protobuf    | 3.20.0           |
+| WasmEdge    | 0.11.2           |
+
+### Using Clang 16.0.0
+
+The XCode that matches MacOS Sequoia 15.3 bundles Clang 16.0.0. It has a performance regression
+when instantiating templates from large enum ranges.
+
+If you experience any hanging compiles, make sure you to merge the following PR:
+https://github.com/Xahau/xahaud/pull/436
```

Comment:
This has been merged, but maybe some old branches, can always remove this later ...